lib/galaxy/datatypes/proteomics.py

```diff
@@ -134,6 +134,12 @@ class MzML(ProteomicsXml):
     blurb = 'mzML Mass Spectrometry data'
     root = "(mzML|indexedmzML)"
 
```

Comment:
can you add another newline here and below ? (https://travis-ci.org/galaxyproject/galaxy/jobs/446615804#L476)